ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml

```diff
@@ -1,7 +1,9 @@
 ---
```

Comment:
It doesn't seem good practice to have default vars in `vars/main.yaml` file... not sure why this was done like this in first place... instead we should get rid of this folder/file and move all the default vars to the new `defaults/main.yml` you created.

Reply:
Moved

Reply:
vars/main.yaml is deleted, but the folder is still there? I have removed entire folder in local, but it doesn't appear in the commit...